### PR TITLE
feat: update version on readme file

### PR DESCRIPTION
**Pull request by Kornfuzion**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# EvolutionSimulator
+# EvolutionSimulator2
```
Comment (brainsmarts) [inline README.md]: Why is it now EvolutionSimulator2?

**Review by brainsmarts — CHANGES_REQUESTED** [review]
Hey I don't think this is ready, address these comments first

**Comment by Kornfuzion** [comment]
Addressed Brian's comments

File: README.md [hunk 2]
```diff
@@ -1,2 +1,5 @@
 # EvolutionSimulator
+
+## Description
+This game allows you to observe the evolution of creatures as they adapt to survive under environmental constraints.
```
Comment (brainsmarts) [inline README.md:4]: I haven't had time to update the read me, thanks!

**Review by brainsmarts — APPROVED** [review]
Looks good to me!

**Pull request merged**